terms.rdf

```diff
@@ -240,6 +286,20 @@
         <rdfs:seeAlso rdf:resource="https://en.wikipedia.org/wiki/Foreign_key" />
     </rdf:Property>
 
+    <rdf:Property rdf:about="http://cldf.clld.org/v1.0/terms.rdf#treeReference"
+                  dc:type="reference-property">
+        <rdfs:comment rdf:parseType="Literal">
+            <p>
+                An identifier referencing a language tree by providing a foreign key into TreeTable
```

Comment:
"An identifier referencing a language tree by providing a foreign key **to** TreeTable." - different preposition? At any rate, '.' at the end is missing.